### PR TITLE
Imported v 0.82 changes

### PR DESCRIPTION
**Pull request by Spiralo-Idioide**
Just replaced all libraries with the ones from the v 0.82 that can be downloaded at the website.
I didn't check it and I'm sure there are things that need not to be merged.

EDIT :
I think I checked everything now. Also I changed 'userHomePath' to 'GP Mod Folder' but that's more some kind of personnal preference.

### CODE REVIEW

File: runtime/lib/version.gp [hunk 1]
```diff
@@ -1 +1 @@
-to libraryVersion { return 'v081' }
+to libraryVersion { return 'v082' }
```
Comment (Spiralo-Idioide) [inline runtime/lib/version.gp:1]: Kept v082 because technically it is the version of the library

File: runtime/lib/Project.gp [hunk 2]
```diff
@@ -685,29 +685,25 @@ method removeClassFromPages Project aClass {
```
Comment (Spiralo-Idioide) [inline runtime/lib/Project.gp]: This one shouldn't be merged but I don't know how to easily get rid of it. Still don't know how to use Git

**Comment by callowaysutton** [comment]
I thought v082 just had a change in the executable

Is this just converting everything so that it says GP Mod instead of GP everywhere?

**Comment by Spiralo-Idioide** [comment]
@callowaysutton 
> I thought v082 just had a change in the executable

I also though that but by using v-081 I realized a huge bug that I had never encountered before when using v-082. 
It's this :
ElseIf

> Is this just converting everything so that it says GP Mod instead of GP everywhere?

No, but I think having a folder called 'GP Mod' might be better.

**Comment by callowaysutton** [comment]
That is weird lol

**Pull request merged**